earth_enterprise/integration_test/test_setup.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+ASSETROOT=/gevol/assets
+SRC_VOLUME=/gevol/src 
+TUTORIAL_PATH=/opt/google/share/tutorials/fusion
+TEST_DATA_PATH=${SRC_VOLUME}/gauge_tests
+export PATH=${PATH}:/opt/google/bin
+
+# Clean up from any previous tests
+./test_teardown.sh
+
+# Script will only download if necessary
+sudo sh ${TUTORIAL_PATH}/download_tutorial.sh
+sudo rm -rf ${TEST_DATA_PATH}/*
```

Comment:
Wonder: This doesn't make sense to me.  We're deleting the `${TEST_DATA_PATH}` directory, and running an `rsync` against it right after that.  How is `rsync` supposed to do anything different than a `cp`, if the `${TEST_DATA_PATH}` directory doesn't exist?